jest.config.js

```diff
@@ -13,6 +13,14 @@ module.exports = {
     'jest-snapshot-serializer-ansi'
   ],
   testEnvironment: 'node',
+  // ignore console warnings in TEST_STANDALONE
+  silent: TEST_STANDALONE,
```

Comment:
For consistency should be this called STANDALONE_TEST perhaps?

Reply:
Variable names have been imported from `prettier`